README.md

```diff
@@ -0,0 +1,68 @@
+# OFirehose
+
+Firehose for the federated social web
+
+## Installation
+
+### Prerequisites
+
+You'll need four things to get started:
+
+* node.js 4.x or higher
+* npm 1.1.0 or higher
+
+### Local install
+
+You need clone the git repository:
+
+    git clone https://github.com/e14n/ofirehose.git
+
+And install the dependencies using `npm`:
+
+    cd ofirehose
+    npm install
+
+### Database setup
+
+OFirehose uses [databank][databank] Any databank driver should work. 
+`couchbase`, `mongodb` and `redis` are probably the best
+bets for production servers, but the `disk` or even `memory` drivers
+can work for testing.
+
+If you're confused, just use the *MongoDB* one, [databank-mongodb](https://www.npmjs.com/package/databank-mongodb).
+
+You can find other drivers like so:
+
+    npm search databank
+
+You can install in the `ofirehose` directory as a normal dependency.
+For example, the databank driver for MongoDB:
+
+    npm install databank-mongodb
+
+Note that you also need to install and configure your database server. 
+example in Debian derivatives: 
+
+    apt-get install mongodb
+
+## Configuration
+
+OFirehose require a configuration file located in `/etc/ofirehose.json`, `~/.ofirehose.json` or `./ofirehose.json`, any of the three options would be fine.
```

Comment:
Not so sure about `./ofirehose.json` - I think we'd be better to just switch to Yargs (#16) and then add a `--config` option like pump.io core has

Reply:
Yes, Yargs is good option